include/asset/asset_strike_price.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+class AssetStrikePrice {
+ public:
+  explicit AssetStrikePrice(const double& strike_price) noexcept;
```

Comment:
Pass by value

> What is “cheap to copy” depends on the machine architecture, but two or three words (doubles, pointers, references) are usually best passed by value.

Ref -> https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const